core-services/secret-mirroring/_mapping.yaml

```diff
@@ -430,29 +430,29 @@ secrets:
     namespace: baremetal
     name: packet-project-id
   to:
-    namespace: test-credentials
+    namespace: cluster-secrets-metal
```

Comment:
You need to also remove all the keys here, otherwise we will refuse to sync: https://github.com/openshift/release/blob/7d13ec291e76c0f85da5d66fd20751add97883a9/core-services/ci-secret-bootstrap/_config.yaml#L1183-L1231

Reply:
The keys in the resulting secret are the keys below `from`

Reply:
The new secrets don't conflict with the old ones. Can I wait until we confirm things work before deleting the old secrets?